classes/Pack.php

```diff
@@ -367,6 +367,13 @@ public static function getItemTable($id_product, $id_lang, $full = false)
                 if (isset($attr_name[0]['id_product_attribute_image']) && $attr_name[0]['id_product_attribute_image']) {
                     $line['id_image'] = $attr_name[0]['id_product_attribute_image'];
                 }
+
+                if (isset($attr_name[0]['attribute_reference'])) {
+                    $line['reference'] = $attr_name[0]['attribute_reference'];
+                } else {
+                    $line['reference'] = '';
+                }
```

Comment:
```suggestion
                $line['reference'] = $attr_name[0]['attribute_reference'] ?? '';
```

Reply:
Hi @Progi1984,
Thanks man ... but this is a PHP 7 feature https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce
and PS 1.7.5 ~ 1.7.6 still support PHP 5.2 for example https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/#php-compatibility-chart

We may be worried about servers in PHP 5.2 🤔 ?

Reply:
Hi @PululuK, 177 version support PHP 7.1 as minimal version. For 178, it's in discussion.

Reply:
I may have misspoke myself ... this PR must work for PS 1.7.5.2 and 1.7.6.4 too, and these versions still support PHP => 5.2.
Suppose I am in a RunTime with a PS 1.7.5.2 and a PHP 5.2 ... Will your suggestion work ?

Reply:
This PR is targeted to `develop` branch, so the future 178 version. Actually, while the issue is not in 176x (patch version) ou 177x (actually in feature freeze) kanban, it can be integrated to previous versions.

Reply:
I'm really sorry ... but how can I tag multiple branches ? Do I have to create a PR for each branch mentioned in the [issue](https://github.com/PrestaShop/PrestaShop/issues/18370)?

Reply:
See on issue with PM. But actually no new issues are integrated to these versions (else major or critical issues)

Reply:
Ok thanks man !